index.html

```diff
@@ -20,6 +43,7 @@ <h2>Guides in other languages</h2>
       <li><a href="https://railspremierspas.humancoders.com/">Guides in French</a></li>
       <li><a href="https://rgcn.github.io">Guides in Chinese</a></li>
       <li><a href="https://rgua.github.io">Guides in Russian</a></li>
+      <li><a href="india.html">Guides in Indian</a></li>
```

Comment:
This should be a link to a fully translated version in another language. In your case I believe it should be Hindi or another language that you want to support.

I have a look into the `india.html` and all I see is an HTML with some objectives written in English and has no translated text of the guides itself.